api/globalSearch/index.ts

```diff
@@ -53,17 +60,17 @@ export async function getSearchResults(req: EnhancedRequest, res: Response, next
  * @param jurisdictions
  * @returns
  */
-export function generateServices(jurisdictions: Jurisdiction[]): GlobalSearchService[] {
+export function generateServices(refDataHMCTS: RefDataHMCTSService[]): GlobalSearchService[] {
   // Retrieve global search services id from config
   const globalSearchServiceIds = getConfigValue(GLOBAL_SEARCH_SERVICES);
   const globalSearchServiceIdsArray = globalSearchServiceIds.split(',');
 
   // Generate global search services
   const globalSearchServices: GlobalSearchService[] = [];
   globalSearchServiceIdsArray.forEach((serviceId) => {
-    const jurisdiction = jurisdictions ? jurisdictions.find((x) => x.id === serviceId) : null;
+    const jurisdiction = refDataHMCTS ? refDataHMCTS.find((x) => x.ccd_service_name === serviceId) : null;
     if (jurisdiction) {
-      globalSearchServices.push({ serviceId: jurisdiction.id, serviceName: jurisdiction.name });
+      globalSearchServices.push({ serviceId: jurisdiction.ccd_service_name, serviceName: jurisdiction.ccd_service_name });
```

Comment:
service name should be set to "jurisdiction.service_short_description"
Need to check that the from end code uses the serviceName field to populate the drop down (not the service Id)

Reply:
I have made the changes as mentioned above